src/MessageTrait.php

```diff
@@ -81,17 +77,17 @@ public function getHeaderLine($header): string
         return \implode(', ', $this->getHeader($header));
     }
 
-    public function withHeader($header, $value): self
+    public function withHeader(string $name, $value): self
```

Comment:
the name should not be changed

Reply:
Ok but it has been changed in psr/http-message v2

Reply:
argument names are not part of the contract in PHP